Runner/main.py

```diff
@@ -0,0 +1,16 @@
+from module.scraper import TweetScraper
+from module.generator import TwitterWordCloud
+
+# create helper objects
+tweet_scraper = TweetScraper()
```

Comment:
If we can comment here what does scrapper is doing then user can easily understand @mratanusarkar